doc/releases/changelog-dev.md

```diff
@@ -216,6 +216,59 @@
          [-0.37133333, -0.18566667,  0.        ,  0.        ,  0.18566667]])
   ```
 
+* The TensorFlow interface now supports devices with shot vectors.
```

Comment:
Let's mention that this is for the new return types specifically.

Reply:
Added clarification.